configs/config.toml.SAMPLE

```diff
@@ -16,7 +16,8 @@
 	# willing to pay for the transaction to be mined. The offered transaction
 	# gas price can not be higher than the max gas price value. If the maximum
 	# allowed gas price is reached, no further resubmission attempts are
-	# performed.
+	# performed.A value can be provided in `wei`, `Gwei` or `ether`, e.g.
```

Comment:
Missing whitespace, but we can fix it later.

Reply:
🤦